src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -152,39 +175,96 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                     <Typography variant="h2" gutterBottom>
                       {t('registration.files_and_license.files')}
                     </Typography>
+
                     {files.length > 0 && (
                       <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mb: '2rem' }}>
-                        {files.map((file) => {
-                          const associatedFileIndex = associatedArtifacts.findIndex((artifact) => {
-                            if (associatedArtifactIsFile(artifact)) {
-                              const associatedFile = artifact as AssociatedFile;
-                              return associatedFile.identifier === file.identifier;
-                            }
-                            return false;
-                          });
+                        <TableContainer component={Paper}>
+                          <Table sx={alternatingTableRowColor}>
+                            <TableHead>
+                              <TableRow>
+                                <TableCell>
+                                  <Typography fontWeight="bold">{t('common.name')}</Typography>
```

Comment:
Dette kan vel fjernes nå, om det virker fra theme
```suggestion
                                  {t('common.name')}
```

Reply:
gjelder resten under her også

Reply:
Funker ikke helt som jeg trodde i theme enda. Er noe funky med import av font, men skal se om jeg finner ut av det.

Reply:
Måtte inn å endre litt på Table likevel, så la inn bold heading i theme i samme slengen, i tillegg til å legge til Paper hvor det ikke var brukt. Så tenker du kan droppe den jobben fra denne PRen, og heller få det fra develop etter det er merget.

https://github.com/BIBSYSDEV/NVA-Frontend/pull/3351